index.js

```diff
@@ -12,6 +12,7 @@ function hasKey(obj, keys) {
 
 function isNumber(x) {
 	if (typeof x === 'number') { return true; }
+	if (!Number.isSafeInteger(Math.floor(x))) { return false; }
```

Comment:
isSafeInteger isn’t available in all the engines we support.

Reply:
Yes. I looked up when `isSafeInteger ` was introduced but then forgot to check the oldest supported version for Minimist, or lack of an explicit minimum. Realised when I saw the test fail.

The PR isn't a no-brainer so I'll only try and make it work with older nodes if otherwise of interest?

Reply:
```suggestion
	if (x >= Math.pow(2, 53)) { return false; }
```
i think this would do it?

My concern is whether this bugfix could arguably be considered a breaking change, in which case it wouldn't likely be worth it.

Reply:
Yes, being conservative it is arguably a breaking change, and it isn't a full solution.